eva/interfaces/relational/db.py

```diff
@@ -86,6 +87,31 @@ def df(self) -> pandas.DataFrame:
             raise Exception("No valid result with the current connection")
         return self._result.frames
 
+    def create_udf(
+        self,
+        udf_name: str,
+        implementation: str,
+        type: str = None,
+        task: str = None,
```

Comment:
Do you think we should make `task` and `model` part of kwargs since they are specific to huggingFace? @jarulraj

Reply:
I think it's a good idea

Reply:
Not relevant to this pr. But I think we can support cachable, batchable, and parallelizable in the kwargs.

Reply:
yes, we should make task and model part of `kwargs` -- `langchain` relies on `kwargs` a lot.

Reply:
addressed in new commit